irs-policy-store/src/test/java/org/eclipse/tractusx/irs/policystore/services/PolicyStoreServiceTest.java

```diff
@@ -79,6 +80,7 @@ class PolicyStoreServiceTest {
 
     private static final String BPN = "testBpn";
     private static final String REGISTER_POLICY_EXAMPLE_PAYLOAD = PolicyStoreControllerTest.REGISTER_POLICY_EXAMPLE_PAYLOAD;
+    private static final String CONFIGURED_DEFAULT_POLICY_ID = "default-policy";
```

Comment:
Why there are different names used for the same policy "default" and "default-policy" this is hard to understand for external contributors. This is some kind of confusing. In case there is a difference between the policy in the configuration and the one in the database the name should reflect exact this cirumstances.

Reply:
@dsmf: This is a test. In the configuration the ID of the default policy can be whatever the admin chooses. "default" is the virtual BPN under which the default policies are stored (because we store policies in MinIO as map of `BPN -> List<Policy>`)